docs/04-operation-guides/troubleshooting/00-eventing/evnt-01-eventing-troubleshooting.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Kyma Eventing - Basic Diagnostics
+---
+
+## Symptom(s)
+
+- You publish an event but the event is not received by the subscriber.
+
+- Subscription is not in ready state.
+
+- You are unable to publish events.
+
+## Remedy
+
+Follow these steps to detect the source of the problem:
+
+### Step 1: Check the status of Check the Eventing backend CR
+
+Check the Eventing backend CR, if the field `EVENTINGREADY` is true:
+   
+```bash
+kubectl -n kyma-system get eventingBackend
```

Comment:
Please let's change `eventingBackend ` to `eventingbackends.eventing.kyma-project.io`.

Reply:
done